platform/osx/display_server_osx.mm

```diff
@@ -273,13 +273,15 @@ - (void)setWindowID:(DisplayServerOSX::WindowID)wid {
 }
 
 - (BOOL)windowShouldClose:(id)sender {
-	ERR_FAIL_COND_V(!DS_OSX->windows.has(window_id), YES);
+	if (!DS_OSX || !DS_OSX->windows.has(window_id))
```

Comment:
Can you add braces around single-statement `if` blocks to fit the new style guide?

Reply:
Done.